modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -65,115 +65,59 @@ endif::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 ifdef::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 . Set the `$RELEASE_IMAGE` variable by running the following command:
 +
-[source,terminal] 
+[source,terminal]
 ----
 $ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
 ----
 endif::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 
+//These conditions could be applied to only what is different (`--cloud=<blah>`) to avoid maintenance difficulties.
 . Extract the list of `CredentialsRequest` objects from the {product-title} release image by running the following command:
 +
 [source,terminal]
 ifdef::aws-sts[]
 ----
 $ oc adm release extract \
---credentials-requests \
---cloud=aws \
---to=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
---from=quay.io/<path_to>/ocp-release:<version>
+  --credentials-requests \
+  --cloud=aws \
+  --to=<path_to_directory_with_list_of_credentials_requests>/credrequests \// <1>
+  --from=quay.io/<path_to>/ocp-release:<version>
```

Comment:
Do we explain `<path_to>` anywhere?  I expect we want something like [this][1] to construct `quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE}`, with maybe some discussion about extracting from `${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${OCP_RELEASE}-${ARCHITECTURE}` in the local-mirroring case?

[1]:  https://github.com/openshift/openshift-docs/blob/1b0517fdb3b389c6f23f160c86bfcfdc8cf6074f/modules/installation-mirror-repository.adoc